src/video/kmsdrm/SDL_kmsdrmsym.h

```diff
@@ -103,7 +103,11 @@ SDL_KMSDRM_SYM(unsigned int,gbm_bo_get_width,(struct gbm_bo *bo))
 SDL_KMSDRM_SYM(unsigned int,gbm_bo_get_height,(struct gbm_bo *bo))
 SDL_KMSDRM_SYM(uint32_t,gbm_bo_get_stride,(struct gbm_bo *bo))
 SDL_KMSDRM_SYM(uint32_t,gbm_bo_get_format,(struct gbm_bo *bo))
+#ifdef SDL_VIDEO_DRIVER_ROCKCHIP
+/* Note: this is not needed if you have ROCKCHIP wrapper for mali */
+#else
 SDL_KMSDRM_SYM(uint32_t,gbm_bo_get_offset,(struct gbm_bo *bo, int plane))
```

Comment:
I guess we don't use this function at all in the current code?

Reply:
This function broke the linkage process if the Rockchip wrapper is not used, so I removed it, just in case. Probably not used.

Reply:
I pushed a commit to the main branch to remove this function since it's unused. You can remove this change from your PR.

Reply:
gbm_bo_get_offset is not used at all, why is there a conflict that must be resolved?

Reply:
ahh, i got it, you just removed it.